PssNationalInstance/src/main/java/com/intellisoft/pssnationalinstance/PssNationalInstanceApplication.java

```diff
@@ -3,16 +3,25 @@
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.context.annotation.PropertySources;
+import org.springframework.context.support.PropertySourcesPlaceholderConfigurer;
 import org.springframework.web.client.RestTemplate;
 import org.springframework.web.reactive.function.client.WebClient;
 
 @SpringBootApplication
+@PropertySources({@PropertySource("classpath:application.properties"), @PropertySource("classpath:.env")})
```

Comment:
The variables in this .env instance take precedence over any other defined elsewhere. Any other env variables defined outside the application will not take effect. 

Ideally, this .env file should be loaded conditionally iff in dev mode. Otherwise, the container's env variables should take precedence.

Reply:
Well noted, this is to mean that the `.env` file should not be in the classpath?